src/pages/index.js

```diff
@@ -90,3 +172,38 @@ editButton.addEventListener('click', ()=>{
 addCardButton.addEventListener('click', ()=>{
   cardModal.open()
 });
+
+avatar.addEventListener('mouseover', ()=>{
+  avatarOverlay.classList.add('profile__photo-overlay_visible')
+})
+
+avatarOverlay.addEventListener('mouseout', ()=> {
+  avatarOverlay.classList.remove('profile__photo-overlay_visible')
+})
+
+avatarOverlay.addEventListener('click', ()=>{
+  changeAvatarModal.open()
+})
+
+//On window load
+api.getCurrentUser().then(data => {
+  userInfo.setUserInfo(data)
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**